source/includes/aks/deploy-minio-on-azure-kubernetes-service.rst

```diff
@@ -51,5 +48,5 @@ For guidance on connecting ``kubectl`` to AKS, see :aks-docs:`Install kubectl an
 Procedure
 ---------
 
-.. include:: /includes/common/common-install-operator-kubectl-plugin.rst
+Something Kustomize
```

Comment:
Hah.

FWIW the procedure should be exactly identical no matter what. EKS, AKS, and GKS should treat `kubectl apply -k github.com/minio/operator` identically.

I can also handle this as part of the deplatformization if we want, but if we go that route we have to push this to an independent branch before we go to main

Reply:
Trying something - same text from k8s (conveniently already an include) as the "Procedure" section for aks, eks, gke. 

I'm not entirely sure the console port forwarding instructions are accurate for not-k8s. Also the admonition about only using Kustomize going forward. I could also copy the include content, minus those bits, into the three other pages.

Or is it close enough, given it's all getting yeeted shortly anyway.

Thoughts? @ravindk89 @djwfyi 

staged:
aks: 
http://192.241.195.202:9000/staging/DOCS-1213-upstream/aks/operations/installation.html#procedure

eks:
http://192.241.195.202:9000/staging/DOCS-1213-upstream/eks/operations/installation.html#procedure

gke:
http://192.241.195.202:9000/staging/DOCS-1213-upstream/gke/operations/installation.html#procedure

Reply:
IMHO this is good enough.